doc/prometheus-metric-schedule.md

```diff
@@ -4,14 +4,14 @@ Prometheus Exports Rules: [Prometheus Exports](./metric-api.md#7prometheusexport
 
 ## Common label
 
-| label name    | Description                                                                                                |
-|:--------------|:-----------------------------------------------------------------------------------------------------------|
-| MetricSubType | The enum MetricSubType value：`DEFAULT,ERROR,CHANNEL,CONSUMER,PRODUCER,CONSUMER_ERROR,PRODUCER_ERROR,NONE` |
-| MetricType    | The Metric Type by metric calculate: `TimerType,HistogramType,MeterType,CounterType,GaugeType`             |
-| host_ipv4     | The ipv4 by host: xxx.xxx.xxx.xx                                                                           |
-| host_name     | host name.                                                                                                 |
-| service       | The `name` read from the configuration. for you service name.                                              |
-| system        | The `system` read from the configuration. for you system name.                                             |
+| label name    |    Value Example    | Description                                                                                                |
+|:--------------|:-------------------:|:-----------------------------------------------------------------------------------------------------------|
+| MetricSubType |      "DEFAULT"      | The enum MetricSubType value: `DEFAULT,ERROR,CHANNEL,CONSUMER,PRODUCER,CONSUMER_ERROR,PRODUCER_ERROR,NONE` |
```

Comment:
All the Value Example can use `value` to replace "Value". use Reverse single quote to replace the quote.

Reply:
as you wish

Reply:
`DEFAULT,ERROR,CHANNEL,CONSUMER,PRODUCER,CONSUMER_ERROR,PRODUCER_ERROR,NONE`
Add more space
`DEFAULT, ERROR, CHANNEL, CONSUMER, PRODUCER, CONSUMER_ERROR, PRODUCER_ERROR, NONE`

Reply:
> `DEFAULT,ERROR,CHANNEL,CONSUMER,PRODUCER,CONSUMER_ERROR,PRODUCER_ERROR,NONE` Add more space `DEFAULT, ERROR, CHANNEL, CONSUMER, PRODUCER, CONSUMER_ERROR, PRODUCER_ERROR, NONE`

You are such a careful man. done!